src/plugins/GamepadManager.js

```diff
@@ -262,24 +262,26 @@ module.exports = class GamepadManager {
      * @param {GamepadEvent} event raw event coming from the browser Gamepad API
      */
     onGamepadConnected(event) {
-        const parsedGamepad = this.parseGamepad(event.gamepad);
-        const customEvent = new CustomEvent('gm-gamepadConnected', {detail: parsedGamepad});
-        if (event.gamepad.mapping === '') {
-            log.error(`Unsupported gamepad mapping for gamepad ${parsedGamepad.name}`);
-            return;
+        if (event.type === 'gamepadconnected') {
+            const customEvent = new CustomEvent('gm-gamepadConnected', {detail: this.parseGamepad(event.gamepad)});
+            window.dispatchEvent(customEvent);
+        } else {
+            log.debug('GamepadManager unknown gamepad connection event');
         }
-        window.dispatchEvent(customEvent);
     }
 
     /**
      * Handler for gamepad disconnection. Also stops listening for inputs for this gamepad and emits a custom event
      * @param {GamepadEvent} event raw event coming from the browser Gamepad API
      */
     onGamepadDisonnected(event) {
```

Comment:
oops I left a typo here
```suggestion
    onGamepadDisconnected(event) {
```